src/modules/screen.c

```diff
@@ -7,7 +7,7 @@ static void receive_waiting_state(const msg_t *msg, UNUSED const void *userdata)
 static int parse_bus_reply(sd_bus_message *reply, const char *member, void *userdata);
 static int get_screen_brightness(void);
 static void timeout_callback(int old_val, bool reset);
-static void pause_screen(bool pause, enum mod_pause type);
+static void pause_screen(bool pause, enum mod_pause type, bool reset_screen_br);
```

Comment:
Enforce the `state.screen_br` resetting when needed, even if screen was already paused.
Fix issues like:
* system going to sleep -> previous code skipped resetting screen br
* while system is sleeping, AC cable is disconnected -> since screen is already paused, screen_br is not resetted